src/types/ChannelOptions.ts

```diff
@@ -1,4 +1,4 @@
-import { Snowflake } from 'discord.js';
+import { Awaitable, GuildMember, PartialGuildMember, Snowflake } from 'discord.js';
```

Comment:
This needs to be reverted.
```suggestion
import { Snowflake } from 'discord.js';
```